<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointSetClassification</class>
 <widget class="QDockWidget" name="PointSetClassification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point set classification</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="create_from_file">
           <property name="text">
            <string>Create item from PLY</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="create_from_item">
           <property name="text">
            <string>Create item from point set item</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Features</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="estimate_parameters">
         <property name="text">
          <string>Estimate parameters</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="gridResolutionLabel">
           <property name="text">
            <string>Grid resolution</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="gridResolutionDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.001000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="radiusNeighborsLabel">
           <property name="text">
            <string>Radius neighbors</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="radiusNeighborsDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.001000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="radiusDTMLabel">
           <property name="text">
            <string>Radius DTM</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="radiusDTMDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.001000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>2.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="compute_features">
           <property name="text">
            <string>Compute features</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="compute_ransac">
           <property name="text">
            <string>Compute RANSAC</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Classification</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="load_config">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_config">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="random_weights">
           <property name="text">
            <string>Try random weights</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="display">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Real colors</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Classification</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Confidence</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Scatter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Planarity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Horizontality</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Elevation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RANSAC</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignBottom">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Class</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="scatterLabel">
           <property name="text">
            <string>Scatter</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="elevationLabel">
           <property name="text">
            <string>Elevation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="horizontalityLabel">
           <property name="text">
            <string>Horizontality</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="planarityLabel">
           <property name="text">
            <string>Non-planarity</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="colorLabel">
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignBottom">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Active</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" alignment="Qt::AlignHCenter">
          <widget class="QDial" name="scatterSlider">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="sliderPosition">
            <number>999</number>
           </property>
           <property name="notchTarget">
            <double>50.000000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3" alignment="Qt::AlignHCenter">
          <widget class="QDial" name="planaritySlider">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="sliderPosition">
            <number>999</number>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="notchTarget">
            <double>50.000000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="4" alignment="Qt::AlignHCenter">
          <widget class="QDial" name="horizontalitySlider">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="sliderPosition">
            <number>999</number>
           </property>
           <property name="notchTarget">
            <double>50.000000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="5" alignment="Qt::AlignHCenter">
          <widget class="QDial" name="elevationSlider">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="sliderPosition">
            <number>999</number>
           </property>
           <property name="notchTarget">
            <double>50.000000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="6" alignment="Qt::AlignHCenter">
          <widget class="QDial" name="colorSlider">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>999</number>
           </property>
           <property name="notchTarget">
            <double>50.000000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="multiply_weights">
           <property name="text">
            <string>Multiply weights</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="run">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Run (quick)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="run_with_smoothing">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Run with smoothing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="run_with_ransac">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Run with RANSAC</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="save">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Save colored point set</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="generate_point_set_items">
           <property name="text">
            <string>Generate point set items</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="extract_2d_outline">
           <property name="text">
            <string>Extract 2D outline</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
